src/explainer/gpt_processing.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+
+import openai
+
+from src.explainer.exceptions import SlideProcessingError
+
+API_KEY = "API_KEY"
```

Comment:
If you want me to put my openai key here you should leave a ToDo here. It's a cool pycharm feature:
https://www.jetbrains.com/help/pycharm/using-todo.html